tracer/test/test-applications/integrations/Samples.OpenTelemetrySdk/Samples.OpenTelemetrySdk.csproj

```
@@ -3,7 +3,7 @@
     <!-- System.Runtime.CompilerServices.Unsafe doesn't support netcoreapp2.1-->
     <!-- https://andrewlock.net/stop-lying-about-netstandard-2-support/-->
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">1.12.0</ApiVersion>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">1.13.1</ApiVersion>
```

Now these tests need a Docker dependency in order to run

```suggestion
        <RequiresDockerDependency>true</RequiresDockerDependency>
        <ApiVersion Condition="'$(ApiVersion)' == ''">1.13.1</ApiVersion>
```